asreview/models/feature_extraction/doc2vec.py

```diff
@@ -90,6 +90,7 @@ class Doc2Vec(BaseFeatureExtraction):
     """
 
     name = "doc2vec"
+    label = "Doc2Vec"
```

Comment:
maybe change to something like `Feature extraction technique provided by the `gensim    &lt;https://radimrehurek.com/gensim/&gt;`__ package`